README.md

```diff
@@ -68,465 +69,304 @@ This approach dramatically reduces latency and reduces load on the OP's servers.
+Note that above claims have to be distinguished from `vp` or `vc` claims as defined in [JWT proof format](https://www.w3.org/TR/vc-data-model/#json-web-token). `vp` or `vc` claims contain those parts of the standard verifiable credentials and verifiable presentations where no explicit encoding rules for JWT exist. They are not meant to include complete verifiable credentials or verifiable presentations objects which is the purpose of the four claims defined in this specification.
 
-### vp_token
+This table shows the different combinations of covered by the claims defined in this specificaiton.
 
-A VP Token is requested by adding a new top level element `vp_token` to the `claims` parameter. This element contains the following sub elements:
-
-`credential_types`
-Object array containing definitions of credential types the RP wants to obtain along with an (optional) definition of the claims from the respective credential type the RP is requesting. Each of those object has the following fields:
+|  | vc_jwt | vp_jwt | vc_ldp | vp_ldp
+|:----------------|:---------------|:---------------|:---------|:--------------------------|
+| Object included in the claim | verifiable credential | verifiable presentation | verifiable credential | verifiable presentation 
+| Proof format on the object| JWT | JWT | LD-Proof | LD-Proof
 
-* `type` - String denoting a credential type
 
-* `claims` - An object determining the claims the RP wants to obtain using the same notation as used underneath `id_token`. 
+# W3C Verifiable Credentials objects returned with JWTs 
+This section illustrates the response when W3C Verifiable Credentials objects are returned with JTWs such as inside ID Token.
 
-`format`
-String designating the VP format. Predefined values are `jwt` and `json-ld`.
+## Self-Issued OP Response
+Below are the examples when W3C Verifiable Credentials are returned inside ID Token as part of Self-Issued OP response. ID Token contains a `vp_jwt` or `vp_ldp` element with the Verifiable Presentation data, or a `vc_jwt` or `vc_ldp` element with the Verifiable Credential data. 
 
-`proof_format`
-[TBD]
+### ID Token with Verifiable Credentials signed as JWTs
 
-Here is an example:
+Below is a non-normative example of ID Token that includes `vp_jwt` claim.
 
-```json
+```
 {
-   "id_token":{
-      "acr":null
-   },
-   "vp_token":{
-      "format":"json-ld",
-      "credential_types":[
-         {
-            "type":"https://www.w3.org/2018/credentials/examples/v1/IDCardCredential",
-            "claims":{
-               "given_name":null,
-               "family_name":null,
-               "birthdate":null
-            }
-         }
-      ]
+  "kid": "did:ion:EiC6Y9_aDaCsITlY06HId4seJjJ...b1df31ec42d0",
+  "typ": "JWT",
+  "alg": "ES256K"
+}.{
+   "iss":"https://book.itsourweb.org:3000/wallet/wallet.html",
```

Comment:
is this a valid SIOP iss claim?

Reply:
self-issued.me is no longer a SIOP requirement?

Reply:
it is, I edited this out.